_pages/en_US/troubleshooting.txt

```diff
@@ -124,19 +124,15 @@ If this is unsuccessful, try using another SD card.
 
 ---
 
-## Installing boot9strap (USM)
+## BannerBomb3
 
-### DSiWare Management menu crashes without showing BB3 multihax menu (USM)
+### DSiWare Management menu crashes without showing BB3 multihax menu (unSAFE_MODE and Fredtool)
```

Comment:
```suggestion
### DSiWare Management menu crashes without showing BB3 multihax menu
```
If we're merging all of them, do we need to specify the exploit?

Reply:
well i thought this works to distinguish the 2, fredtool and unsafe mode, and Bannerbomb+Fredtool (TWN), but we don't need to